src/bootstrap/src/core/build_steps/check.rs

```diff
@@ -386,7 +386,7 @@ impl Step for RustAnalyzer {
             cargo_subcommand(builder.kind),
             "src/tools/rust-analyzer",
             SourceType::InTree,
-            &["rust-analyzer/in-rust-tree".to_owned()],
+            &["in-rust-tree".to_owned()],
```

Comment:
I changed these prefixed ones given we didn't prefix the `sysroot-abi` feature either in the other places, and I was kind of confused that these worked in the first place